docs/content/content-blocks.md

```diff
@@ -130,6 +130,8 @@ My content
 ````
 `````
 
+You can also [create custom admonitions using custom CSS](custom-admonitions).
+
```

Comment:
```suggestion
```

I think at least for now, we shouldn't drive too much attention to this practice since it may be unstable for people. In the future we can revisit if we can nail down a more dependable pattern that won't change with updates to the dependencies.

Reply:
that makes sense to me.